README.md

```diff
@@ -84,6 +84,7 @@ Chapter 2 - Names and values
 
 - Cohort 1: [Asmae Toumi](https://twitter.com/asmae_toumi) ([slides](https://r4ds.github.io/bookclub-Advanced_R/Presentations/Week02/Cohort1/Chap2slides.html#1))
 - Cohort 2: [Josh Pohlkamp-Hartt](https://twitter.com/jpohlkamphartt) ([slides](https://r4ds.github.io/bookclub-Advanced_R/Presentations/Week02/Cohort2_America/Chapter2Slides.html#1) | video)
+- Cohort 3: [Anna-Leigh Brown](https://twitter.com/annaleighbrown2)([video](https://youtu.be/-bEXdOoxO_E))
```

Comment:
Were the slides made in a ~powerpoint-like system? Might be helpful to upload those, but if not PDF is fine. You can link it with 

```
[slides](https://github.com/r4ds/bookclub-Advanced_R/blob/main/Presentations/Week02/Cohort3_Africa_Europe/Advanced%20R%20Chapter%202%20Names%20and%20values.pdf)

Reply:
Made in Google Docs, happy to link directly to Google Docs rather than PDF